exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestJsonReader.java

```diff
@@ -81,6 +81,8 @@ public void testEmptyList() throws Exception {
 
   @Test
   public void schemaChange() throws Exception {
+    // TODO: What is this testing? Presumably, we are simply
```

Comment:
I assume yes, it just tests that test pass, do we need this comment here though?

Reply:
Revised the comment.